docker/start_celery.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
-$(cd dirname $0/..; C_FORCE_ROOT=true /venv/spacedock/bin/celery -A KerbalStuff.celery worker --loglevel=info)
+cd $(dirname $0/)
```

Comment:
Wow, that's quite a difference! :astonished:

Reply:
I actually don't know what that `cd` is for since the path to `celery` is absolute (and I don't know anything about celery),
but I _do_ know that how it was before is an definite syntax error.

Reply:
Maybe it reads configs or writes logs in the current dir? Either way, that `cd` command wasn't doing anything sensible.

Reply:
I'm not sure if it is doing something sensible now, but at least it works :smile: